ci/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+# You can run it from any directory.
+PROJECT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
+
+"$PROJECT_DIR"/gradlew --no-daemon --info clean build
```

Comment:
Replace `--info` with `--stacktrace`?

Reply:
They're not the same though, `--stacktrace` doesn't increase log level, it just prints stacktrace in case of it, but lemme add it

Reply:
BTW, check the full log, seems like `./gradlew assemble` is executed for some reason before `build.sh`.

Reply:
But do we need increased log level?

Reply:
But do we need `--stacktrace`? It will print stacktrace even if test fails for example :)

It's typical problem — you don't need too much logs unless something went wrong